icons/index.ts

```diff
@@ -44,6 +44,7 @@ import { AttachFileIcon } from '@/icons/attach-file';
 import { GaugeIcon } from '@/icons/gauge';
 import { MenuIcon } from '@/icons/menu';
 import { ClockIcon } from '@/icons/clock';
+import { AlarmClock } from '@/icons/alarm-clock';
```

Comment:
```suggestion
import { AlarmClockIcon } from '@/icons/alarm-clock';
```

Reply:
I've made the requested changes. Please review when you have a moment.

Reply:
thanks, and could you please fix merge conflicts